### ISSUE TITLE
[bug] Poor performance when window set to floating in i3wm

### ISSUE THREAD

**Issue by pplanel** [tauri-apps/tauri#4077]
### Describe the bug

In i3wm if the window is set to float `i3-msg [class="main"] floating enable` the contents of the window became slow and unresponsive (selecting text, resizing, clicking on elements).

### Reproduction

1. Run any example in tauri repo
2. Get window _WM_CLASS with `xprop`
3. `i3-msg [class="$window_class"] floating enable`
4. Interact with window

### Expected behavior

Window content should behave correctly.

### Platform and versions

```shell
Environment
   › OS: NixOS 21.11.0 X64
   › Node.js: 16.15.0
   › npm: 8.5.5
   › pnpm: 7.0.0
   › yarn: 1.22.18
   › rustup: 1.24.3
   › rustc: 1.60.0
   › cargo: 1.60.0
   › Rust toolchain: stable-x86_64-unknown-linux-gnu

 Packages
   › @tauri-apps/cli [NPM]: 1.0.0-rc.9
   › @tauri-apps/api [NPM]: 1.0.0-rc.4
   › tauri [RUST]: 1.0.0-rc.8,
   › tauri-build [RUST]: 1.0.0-rc.7,
   › tao [RUST]: 0.8.4,
   › wry [RUST]: 0.15.1,

 App
   › build-type: bundle
   › CSP: default-src 'self'
   › distDir: ../frontend/dist
   › devPath: http://localhost:8080/

 App directory structure
   ├─ public
   ├─ dist
   ├─ src
   ├─ node_modules
   └─ target
```
```


### Stack trace

_No response_

### Additional context

_No response_

**Comment by aarchangel64** [comment]
I can reproduce this issue (I'm directly using Wry instead of the full Tauri stack).

**Comment by amrbashir** [comment]
Does the same happen with epiphany browser?

**Comment by aarchangel64** [comment]
On further testing, I can't actually recreate this issue - my sincere apologies!

I ran Wry's hello_world.rs example, replacing the URL with [this benchmark](https://browserbench.org/Speedometer2.0/).

The results are as follows:
- Vivaldi Browser (Chromium): 133 ± 1.5
- Wry (i3 Tiled Mode): 178 ± 3.0
- Wry (i3 Floating Mode): 182 ± 2.4

On different note, I've noticed that CSS seem animations / transitions to be _much_ slower than in Chromium (at least on Linux).
Using [this benchmark](https://greensock.com/js/speed.html), with the engine set to CSS Transitions and number of Dots set to 50, I get these results:

- Vivaldi Browser (Chromium): ~100 FPS
- Epiphany Browser: ~14 FPS
- Wry (i3 Tiled Mode): ~14 FPS
- Wry (i3 Floating Mode): ~14 FPS

Should I open a new issue for this? I'm not sure, considering it seems to be an issue with WebKit, not Tauri / Wry.
